### PR TITLE
[clang-format][NFC] Drop "Always" in "AlwaysBreakAfterReturnType".

### PR DESCRIPTION
**Pull request by rmarker**
Complete the switch from "AlwaysBreakAfterReturnType" to "BreakAfterReturnType".

### CODE REVIEW

**Comment by rmarker** [comment]
@owenca, here is a pull request to complete the renaming of the option.
Following on from #80827, in the style of [7664ddf](https://github.com/llvm/llvm-project/commit/7664ddf8811242295abb837640cad8dd8cefb5e8).

File: clang/lib/Format/Format.cpp [hunk 1]
```diff
@@ -1871,10 +1869,10 @@ FormatStyle getWebKitStyle() {
 FormatStyle getGNUStyle() {
   FormatStyle Style = getLLVMStyle();
   Style.AlwaysBreakAfterDefinitionReturnType = FormatStyle::DRTBS_All;
-  Style.AlwaysBreakAfterReturnType = FormatStyle::RTBS_AllDefinitions;
   Style.BreakBeforeBinaryOperators = FormatStyle::BOS_All;
   Style.BreakBeforeBraces = FormatStyle::BS_GNU;
   Style.BreakBeforeTernaryOperators = true;
+  Style.BreakAfterReturnType = FormatStyle::RTBS_AllDefinitions;
```
Comment (owenca) [inline clang/lib/Format/Format.cpp:1875]: Not sorted.

File: clang/lib/Format/Format.cpp [hunk 2]
```diff
@@ -1130,14 +1129,13 @@ template <> struct MappingTraits<FormatStyle> {
     // AlwaysBreakAfterReturnType was not, initialize the latter from the
     // former for backwards compatibility.
```
Comment (owenca) [inline clang/lib/Format/Format.cpp:1130]:
```suggestion
    // BreakAfterReturnType was not, initialize the latter from the former for
    // backwards compatibility.
```

File: clang/lib/Format/Format.cpp (hunk 1 above)
Comment (rmarker) [reply clang/lib/Format/Format.cpp]: Done.

File: clang/lib/Format/Format.cpp (hunk 2 above)
Comment (rmarker) [reply clang/lib/Format/Format.cpp]: Done.

**Review by owenca — APPROVED** [review]

**Pull request merged**